notebooks/tools/terraform/instance.tf

```diff
@@ -0,0 +1,66 @@
+resource "random_id" "instance_id" {
+  byte_length = 8
+}
+
+resource "google_compute_instance" "default" {
+  name         = "vm-${random_id.instance_id.hex}"
```

Comment:
My only concern is that this be stable enough to tear down and be updated.  Perhaps putting it in a local first, but I need to look that up.